src/components/Registration/BusinessTypeConfirm.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div id="business-type-confirm">
+    <v-row no-gutters>
+      <v-col cols="12" sm="3" class="pr-4">
+        <label class="d-block title-label">Business Type</label>
+        <v-chip
+          v-if="hasBusinessTypeChecked"
+          id="checked-chip"
+          x-small label
+          color="primary"
+          text-color="white"
+        >
+          Checked
+        </v-chip>
+      </v-col>
+
+      <v-col cols="12" sm="9">
+        <p class="mb-0">General Partnership</p>
+        <div id="business-check-div">
+          <v-checkbox
+            class="mt-0"
+            v-model="checked"
+            hide-details
+            label="I acknowledge that a General Partnership cannot be changed into a
+            Sole Proprietorship (including DBA). If this is necessary, a new
+            Name Request Number and Statement of Registration (along with
+            associated fees) will be required."
```

Comment:
In commit ac78c6c, I removed the p tag and used label instead. The checkbox is now aligned with the label top.